test/run.js

```diff
@@ -36,6 +36,7 @@ function build(type, features) {
   // sh.exec('git commit -nm init')
   // sh.exec('git checkout -B origin/master')
   process.env = npmPath.env({env: process.env})
+  sh.exec('yarn prepack')
```

Comment:
Since tests are now ran from the code that's compiled and not from the in-memory compilations we need to run `prepack` before running `test`.